<ui version="4.0" >
 <class>ECCDLOGIN</class>
 <widget class="QDialog" name="ECCDLOGIN" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>20</y>
     <width>290</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Please enter your password.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>46</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMask" >
    <string/>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="maxLength" >
    <number>12</number>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>50</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>100</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>100</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>CANCLE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
